### PR TITLE
fixing IDENTITY-3773

### PR DESCRIPTION
**Pull request by madurangasiriwardena**

### CODE REVIEW

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java [hunk 1]
```diff
@@ -1076,23 +1076,23 @@ public IdentityProvider getIdPByAuthenticatorPropertyValue(Connection dbConnecti
...
 
-                federatedIdp.setHomeRealmId(rs.getString("idp.HOME_REALM_ID"));
-                federatedIdp.setCertificate(getBlobValue(rs.getBinaryStream("idp.CERTIFICATE")));
+                federatedIdp.setHomeRealmId(rs.getString("HOME_REALM_ID"));
+                federatedIdp.setCertificate(getBlobValue(rs.getBinaryStream("CERTIFICATE")));
                 federatedIdp.setAlias(rs.getString("idp.ALIAS"));
 
                 JustInTimeProvisioningConfig jitProConfig = new JustInTimeProvisioningConfig();
-                if (rs.getString("idp.INBOUND_PROV_ENABLED").equals("1")) {
+                if (rs.getString("INBOUND_PROV_ENABLED").equals("1")) {
```
Comment (pulasthi7) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: flip "equals()"

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java [hunk 2]
```diff
@@ -511,17 +511,17 @@ private ClaimConfig getIdPClaimConfiguration(Connection dbConnection, String idP
...
 
                 claimMapping.setLocalClaim(localClaim);
                 claimMapping.setRemoteClaim(idpClaim);
-                claimMapping.setDefaultValue(rs.getString("IDP_CLAIM_MAPPING.DEFAULT_VALUE"));
-                if (("1").equals(rs.getString("IDP_CLAIM_MAPPING.IS_REQUESTED"))) {
+                claimMapping.setDefaultValue(rs.getString("DEFAULT_VALUE"));
+                if (("1").equals(rs.getString("IS_REQUESTED"))) {
                     claimMapping.setRequested(true);
-                } else if (("0").equals(rs.getString("IDP_CLAIM_MAPPING.IS_REQUESTED"))) {
+                } else if (("0").equals(rs.getString("IS_REQUESTED"))) {
```
Comment (pulasthi7) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: redundant "()" for "0"

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java [hunk 3]
```diff
@@ -1076,23 +1076,23 @@ public IdentityProvider getIdPByAuthenticatorPropertyValue(Connection dbConnecti
...
 
-                idpId = rs.getInt("idp.ID");
-                idPName = rs.getString("idp.NAME");
+                idpId = rs.getInt("ID");
+                idPName = rs.getString("NAME");
 
                 federatedIdp.setIdentityProviderName(idPName);
 
-                if (("1").equals(rs.getString("idp.IS_PRIMARY"))) {
+                if (("1").equals(rs.getString("IS_PRIMARY"))) {
```
Comment (pulasthi7) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: redundant "()" for "1"

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java (hunk 2 above)
Comment (madurangasiriwardena) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Fixed with 5eb8104dee8753dd68e80a010a852b92614bad0c

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java (hunk 3 above)
Comment (madurangasiriwardena) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Fixed with 5eb8104dee8753dd68e80a010a852b92614bad0c

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java (hunk 1 above)
Comment (madurangasiriwardena) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java]: Fixed with https://github.com/wso2/carbon-identity/pull/1679

**Pull request merged**